frontend/src/components/tasks/createTask.jsx

```diff
@@ -23,8 +23,7 @@ let CreateTask = React.createClass({
         name: '',
         description: '',
         score: 1,
-        userId: null,
-        sprintId: this.props.sprintId || null
```

Comment:
According to React, having `this.props` used in `getInitialState` is an antipattern. The reason being that changes in `this.props` won't flow through to the `state`, resulting in multiple sources of truths. What was happening was, this `createTask` component was being initialized with `null` passed into `this.props.sprint`, but the parent state would change (as a result of fetching the current sprint), and it would update `this.props.sprint`, but _not_ `this.state`